Scenarios/Authentication/Mixed-auth-allow-anonymous-workaround/Idp/Program.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Claims;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.IdentityModel.Tokens;
+using OpenIddict.Abstractions;
+using OpenIddict.Server;
+using OpenIddict.Server.AspNetCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddOpenIddict()
+    .AddServer(options =>
+    {
+        options.AddDevelopmentEncryptionCertificate();
+        options.AddDevelopmentSigningCertificate();
+        options.AllowClientCredentialsFlow();
+        options.SetTokenEndpointUris("/token");
```

Comment:
You may want to remove the leading slash from the token endpoint URI for the reasons explained here: https://github.com/openiddict/openiddict-core/issues/1613

Reply:
done thanks